docs/source/technical_tutorials/searching.rst

```diff
@@ -2,147 +2,15 @@
 Search
 =======================================
 
-MAAP's Dual Catalog
```

Comment:
I think it's fine for this whole section on Dual Catalog to stay on the Search landing page.

Reply:
I think the issue with having the dual catalog content on the searching.rst page is then only those headers (the ones in the dual catalog documentation show up, not the other pages like "search using EDSC". I think we want the other pages to show up in the left hand navigation, so I'm going to keep the dual catalog content on a separate page. But I have modified the text in the top section of searching.rst to be a bit more descriptive as you suggested in the other comment on this PR.